setuptools_scm/utils.py

```diff
@@ -32,9 +34,7 @@ def _always_strings(env_dict):
     On Windows and Python 2, environment dictionaries must be strings
     and not unicode.
     """
-    is_windows = platform.system == 'Windows'
```

Comment:
By the way, `platform.system()` is a function, not a static string attribute; thus the comparison `platform.system == 'Windows'` always returned False there.

https://docs.python.org/3/library/platform.html#platform.system